### ISSUE TITLE
New failing test: geoid/dynamic_topography.00000

### ISSUE THREAD

**Issue by bangerth** [geodynamics/aspect#1590]
It looks like all pull requests current fail this test:

```
156/447 Test #158: geoid ..........................................................***Failed    0.17 sec
*** "Test geoid failed": ***
[1/2] cd /home/dealii/build-gcc/tests && /usr/bin/perl -pi /home/dealii/source/tests/normalize.pl `find /home/dealii/build-gcc/tests/output-geoid/ -type f`
[2/2] Generating output-geoid/dynamic_topography.00000.diff
FAILED: output-geoid/dynamic_topography.00000.diff 
cd /home/dealii/build-gcc/tests && /home/dealii/source/tests/cmake/diff_test.sh /usr/bin/numdiff geoid/dynamic_topography.00000 /home/dealii/source/tests /home/dealii/build-gcc/tests
******* Error during diffing output results for geoid/dynamic_topography.00000
******* Results are stored in /home/dealii/build-gcc/tests/output-geoid/dynamic_topography.00000.diff.failed
******* Check /home/dealii/build-gcc/tests/output-geoid/dynamic_topography.00000 /home/dealii/source/tests/geoid/dynamic_topography.00000
******* All of 12 lines of diffs are:
----------------
##1       <== # x y z topography
##1       ==> # x y z upper surface topography

##1       #:5   <== topography
##1       #:5   ==> upper
@                                                     @@
##1             <==
##1       #>6   ==> surface topography
@ Line 1 in file "/home/dealii/build-gcc/tests/output-geoid/dynamic_topography.00000.cmp.notime" is shorter than expected!

+++  File "/home/dealii/build-gcc/tests/output-geoid/dynamic_topography.00000.cmp.notime" differs from file "/home/dealii/build-gcc/tests/output-geoid/dynamic_topography.00000.notime"
ninja: build stopped: subcommand failed.

CMake Error at /home/dealii/source/tests/run_test.cmake:14 (MESSAGE):
  *** Test aborted.
```

@Shangxin-Liu -- can you see if you can fix this?

**Comment by Shangxin-Liu** [comment]
Yeah, just one line comment because I submitted my pull request which uses the previous dynamic topography.cc to calculate grid dynamic topography before Jacky's new now is merged. I already committed a patch to fix this issue. @bangerth

**Comment by gassmoeller** [comment]
Fixed by #1585
